doc/userguide/upgrade.rst

```diff
@@ -30,6 +30,19 @@ by the ones Suricata supplies.
 Major updates include new features, new default settings and often also
 remove features.
 
+Upgrading 6.0 to 7.0
+--------------------
+- Protocol values and their names are built-in to Suricata instead of using the system's ``/etc/protocols`` file. Some names and casing may have changed.
+
+Major changes
+~~~~~~~~~~~~~
+
+Removals
+~~~~~~~~
+
+Logging changes
+~~~~~~~~~~~~~~~
+
```

Comment:
I'm OK with this just under the `Upgrading 6.0 to 7.0` heading.. But if we're going to add a logging section now, should it not go there instead?

Reply:
Ok ... I'll move the note to the logging changes section.